floors/floors.go

```diff
@@ -71,29 +71,34 @@ func updateBidRequestWithFloors(extFloorRules *openrtb_ext.PriceFloorRules, requ
 	if len(modelGroup.Values) > 0 {
 		for _, imp := range request.GetImp() {
 			desiredRuleKey := createRuleKey(modelGroup.Schema, request, imp)
-			matchedRule, isRuleMatched := findRule(modelGroup.Values, modelGroup.Schema.Delimiter, desiredRuleKey, len(modelGroup.Schema.Fields))
-
+			matchedRule, isRuleMatched := findRule(modelGroup.Values, modelGroup.Schema.Delimiter, desiredRuleKey)
+			floorVal = modelGroup.Default
```

Comment:
Added changes to consider default value if matching rule is not found. 
Reference:  https://docs.google.com/document/d/1YAt00RfQaU2Ugciqw1LnirLgjyUz9ibfyjfxWUHR_P0/edit?disco=AAAArinhqdc